tsconfig.json

```diff
@@ -86,5 +86,6 @@
       /* Advanced Options */
       "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
     },
-    "exclude": ["dist", "node_modules", "test", "webpack.config.js"]
+    "exclude": ["dist", "node_modules", "test", "webpack.config.js"],
```

Comment:
"exclude": ["dist", "node_modules", "webpack.config.js"]

Reply:
@m0cah one line is still not correct

Reply:
@m0cah please remove the "test" from the exclude array (we want to include the test files for typechecking" webpack will exclude them when it does the build for prod

Reply:
Hey Josh, sorry for the delay. Should be fixed now.